pkgs/development/python-modules/django-debug-toolbar/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib, fetchFromGitHub
+
+# Build
+, buildPythonPackage
+
+# Runtime
+, django
+, jinja2
+, sqlparse
+
+# Test
+, coverage
+, html5lib
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "django-debug-toolbar";
+  version = "3.2.2";
+
+  src = fetchFromGitHub {
+    owner = "jazzband";
+    repo = pname;
+    rev = version;
+    sha256 = "1dgb3s449nasbnqd5xfikxrfhwwilwlgrw9nv4bfkapvkzpdszjk";
+  };
+
+  propagatedBuildInputs = [
+    django
+    jinja2
+    sqlparse
+  ];
+
+  DB_BACKEND = "sqlite3";
+  DB_NAME = ":memory:";
+  TEST_ARGS = "tests";
+  DJANGO_SETTINGS_MODULE = "tests.settings";
+
+  checkInputs = [
+    coverage
+    html5lib
+    python
+  ];
+
+  checkPhase = ''
+    runHook preCheck
+
+    # test
+    ${python.interpreter} -m django test ${TEST_ARGS}
+
+    # coverage
+    ${python.interpreter} --version
+    ${python.interpreter} -b -W always -m coverage run -m django test -v2 ${TEST_ARGS}
```

Comment:
```suggestion
    ${python.interpreter} -b -W always -m django test -v2 ${TEST_ARGS}
```

Reply:
i don't understand why to remove the coverage test, but i removed it altogether anyway.

Reply:
We can't upload the coverage results and generating them takes resources so the general consensus is to not generate them in the first place.

Reply:
Not sure what you mean with upload here.

The reason we don't do coverage is that as a metric we don't care for it. Most of the time we are in no position to change coverage of upstream projects.